### PR TITLE
add src_rsv_addr and dst_rsv_addr to trade log

### PR DESCRIPTION
**Pull request by cyberliem**

### CODE REVIEW

**Comment by cyberliem** [comment]
[Add source_reserve_address/dest_reserve_address to trades measurement](https://trello.com/c/nT3JI4qM/293-add-sourcereserveaddress-destreserveaddress-to-trades-measurement)

File: tradelogs/storage/influx.go [hunk 1]
```diff
@@ -186,6 +186,8 @@ func (is *InfluxStorage) tradeLogToPoint(log common.TradeLog, rate tokenrate.ETH
 		"ip":      log.IP,
 
 		"eth_rate_provider": rate.Provider,
+		"src_rsv_addr":      log.SrcRsvAddress.String(),
```
Comment (namnm1991) [inline tradelogs/storage/influx.go]: For our purpose of this issue, I think we only need to add src_rsv_addr, dst_rsv_addr into measurement "trades" here. No need to add these new field into TradeLog struct.
The logic to prepare for these two tag is as follow:
token -> token: src_rsv_addr = log.burnFees[0].ReserveAddress, dst_rsv_addr=log.burnFee[1].ReserveAddress
token-> eth: src_rsv_addr =  log.burnFees[0].ReserveAddress
eth -> token: dst_rsv_addr = log.burnFees[0].ReserveAddress

File: tradelogs/crawler.go [hunk 2]
```diff
@@ -39,7 +39,7 @@ const (
 	internalNetworkAddr = "0x91a502C678605fbCe581eae053319747482276b9"
 
 	//ethDecimals int64  = 18
-	//ethAddress  string = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"
+	// ethAddress string = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"
```
Comment (favadi) [inline tradelogs/crawler.go]: please remove thoses unused consts.

File: tradelogs/storage/influx.go [hunk 3]
```diff
@@ -15,6 +15,10 @@ import (
 	"github.com/KyberNetwork/reserve-stats/tradelogs/common"
 )
 
+const (
+	ethAddress string = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"
```
Comment (favadi) [inline tradelogs/storage/influx.go]: this address is available in lib/blockchain.ETHAddr already.

File: tradelogs/storage/testdata/trade_logs.json [hunk 4]
```diff
@@ -19,7 +19,9 @@
         ],
         "wallet_fees": null,
         "ip": "",
-        "country": ""
+        "country": "",
+        "dst_rsv_addr" : "0x63825c174ab367968ec60f061753d3bbd36a0d8f",
+        "src_rsv_adrr" : "0x0000000000000000000000000000000000000000"
```
Comment (favadi) [inline tradelogs/storage/testdata/trade_logs.json]: those fields should be removed?

**Pull request merged**